test/gtest/stringify.cpp

```diff
@@ -254,7 +254,7 @@ std::vector<SuffixTree::RepeatedSubstring>
 repeatSubstrings(std::vector<uint32_t> hashString) {
   SuffixTree st(hashString);
   std::vector<SuffixTree::RepeatedSubstring> substrings =
-    std::vector(st.begin(), st.end());
+    std::vector<SuffixTree::RepeatedSubstring>(st.begin(), st.end());
```

Comment:
Even better, you could just do `std::vector<SuffixTree::RepeatedSubstring> substrings(st.begin(), st.end());`.

Reply:
done